docs/src/modules/components/ApiPage/list/ExpandableApiItem.tsx

```diff
@@ -56,13 +56,13 @@ const Root = styled('div')<{ ownerState: { type?: DescriptionType } }>(
       p: { marginBottom: theme.spacing(1.5) },
     },
     '& .MuiApi-item-note': {
-      fontSize: 12,
       marginLeft: 2,
-      letterSpacing: '1px',
-      textTransform: 'uppercase',
       color: `var(--muidocs-palette-success-800, ${lightTheme.palette.success[800]})`,
+      fontSize: theme.typography.pxToRem(10),
```

Comment:
Although there's no definition of a minimum font-size, I'd avoid going smaller than `12px`—with this, we could also remove the `text-transform`.

Regardless, I think this is a perfect use case for a `chip`, what do you think?

Reply:
I've bumped it up to 11 to ease it a bit — the monospace font makes 12px look much bigger! I agree with using a Chip, but these subtitles/note things are set up in a way where we're toggling whether it renders as green or yellow using classes, and to do that with a Chip (where I'd need to toggle either the `color` prop or using the sx somehow) would get a bit annoying 😖